scripts/west_commands/runners/dediprog.py

```diff
@@ -54,6 +54,9 @@ def do_run(self, command, **kwargs):
             cmd_flash.append('--vcc')
             cmd_flash.append(self.vcc)
 
+        # Allow to flash images smaller than flash device capacity
+        cmd_flash.append('-x')
+        cmd_flash.append('0')
```

Comment:
Why not FF?  I don't understand why that wouldn't work and it should reduce flash wear if it can work.

Reply:
That works too. Changed to reduce wear as indicated.